yarn.lock

```diff
@@ -2751,6 +2778,31 @@
   dependencies:
     "@sinonjs/commons" "^2.0.0"
 
+"@spotlightjs/overlay@^0.0.17":
+  version "0.0.17"
+  resolved "https://registry.yarnpkg.com/@spotlightjs/overlay/-/overlay-0.0.17.tgz#d3855148937a4fa6485ac478b46f52bd88d83cf7"
+  integrity sha512-zBcPwAMDG2hbrPKFShSl3yukGliXTScnY3e7hu3CpvvvylAA5isc8gOINVBz3AlRlf3EquVByWk+T+yxg2QlAw==
+  dependencies:
+    "@fontsource/raleway" "^5.0.8"
+    "@sentry/utils" "^7.77.0"
+    autoprefixer "^10.4.15"
+    dayjs "^1.11.9"
+    platformicons "^5.6.2"
+    postcss "^8.4.28"
+    react "^18.2.0"
+    react-dom "^18.2.0"
```

Comment:
having react 18 and 17 is concerning. this should be a peer dependency with supported versions

Reply:
This shouldn't require a dep at all and we certainly wouldn't want to try to support multiple React versions. Any idea what we're doing wrong here @HazAT?

Like this isnt an ESM module right? Its fully compiled/bundled and already includes React?